### ISSUE TITLE
Breast intraductal papilloma

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7952]
**Preferred term label:**
Breast intraductal papilloma

**Synonyms**

**Definition (free text, please give PubMed ID)**
Intraductal papilloma is a benign tumor found within breast ducts. The abnormal proliferation of ductal epithelial cells causes growth. A solitary intraductal papilloma is usually found centrally posterior to the nipple affecting the central duct. Multiple intraductal papillomas are located peripherally, found in any breast quadrant affecting the peripheral ducts.
PMID:30137824
NCIT:C3863

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
617100 
PMID: 27476653    2/3 

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000102
- label: Breast intraductal papilloma
- parents: [Neoplasm of the breast (HP:0100013)](https://hpo.jax.org/app/browse/term/HP:0100013)
- pmid: PMID:30137824
- def: Intraductal papilloma is a benign tumor found within breast ducts. The abnormal proliferation of ductal epithelial cells causes growth. A solitary intraductal papilloma is usually found centrally posterior to the nipple affecting the central duct. Multiple intraductal papillomas are located peripherally, found in any breast quadrant affecting the peripheral ducts.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7952
- synonyms:
